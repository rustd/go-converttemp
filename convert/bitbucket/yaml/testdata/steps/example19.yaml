# Example using the download option to prevent artifacts from downloading in a step

pipelines:
  default:
    - step:
        name: Build and test
        script:
          - npm install
          - npm run build
        artifacts: # Store build artifacts for use in the following steps
          - dist/**
    - step:
        name: lint code and store results
        script:
          - npm lint > results.txt
        artifacts:
          download: false # Block artifacts downloading, they're not needed for this step
    - step:
        name: Test code from build step stored in the dist/ directory
        script:
          - npm test
