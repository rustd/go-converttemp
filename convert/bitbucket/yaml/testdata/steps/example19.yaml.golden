{
    "pipelines": {
        "default": [
            {
                "step": {
                    "name": "Build and test",
                    "script": [
                        "npm install",
                        "npm run build"
                    ],
                    "artifacts": [
                        "dist/**"
                    ]
                }
            },
            {
                "step": {
                    "name": "lint code and store results",
                    "script": [
                        "npm lint > results.txt"
                    ],
                    "artifacts": {
                        "download": false
                    }
                }
            },
            {
                "step": {
                    "name": "Test code from build step stored in the dist/ directory",
                    "script": [
                        "npm test"
                    ]
                }
            }
        ]
    }
}